From dfa75cee07f49cad8fbd4ab5ef75f517f3cddff8 Mon Sep 17 00:00:00 2001
From: Jingchang Lu <jingchang.lu@freescale.com>
Date: Thu, 12 Jun 2014 15:38:17 +0800
Subject: [PATCH 195/255] arm: dts: ls1021a: add enet dma transfer endian
 indicator

Signed-off-by: Jason Jin <Jason.Jin@freescale.com>
Signed-off-by: Jingchang Lu <jingchang.lu@freescale.com>
[Kevin: Original patch taken from
LS1021A-SDK-V1.1-ARM-SOURCE-20140815-yocto.iso]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index 4f0b2f6..429ffdb 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -625,6 +625,7 @@
 			device_type = "network";
 			model = "eTSEC";
 			compatible = "fsl,etsec2";
+			fsl,dma-endian-le;
 			fsl,num_rx_queues = <0x1>;
 			fsl,num_tx_queues = <0x1>;
 			local-mac-address = [ 00 00 00 00 00 00 ];
@@ -650,6 +651,7 @@
 			device_type = "network";
 			model = "eTSEC";
 			compatible = "fsl,etsec2";
+			fsl,dma-endian-le;
 			fsl,num_rx_queues = <0x1>;
 			fsl,num_tx_queues = <0x1>;
 			local-mac-address = [ 00 00 00 00 00 00 ];
@@ -675,6 +677,7 @@
 			device_type = "network";
 			model = "eTSEC";
 			compatible = "fsl,etsec2";
+			fsl,dma-endian-le;
 			fsl,num_rx_queues = <0x1>;
 			fsl,num_tx_queues = <0x1>;
 			local-mac-address = [ 00 00 00 00 00 00 ];
-- 
2.0.2

